modules/reference/pages/cluster-properties.adoc

```diff
@@ -293,6 +293,69 @@ Max number of simultaneous connections to S3 per shard. Includes connections use
 
 ---
 
+=== cloud_storage_recovery_topic_validation_mode
+
+Validation performed before recovering a topic from cloud storage. In case of failure, the reason for the failure appears as `ERROR` lines in the Redpanda application log. For each topic, reports errors for all partitions, but for each partition, only the first error is reported.
```

Comment:
```suggestion
Validation performed before recovering a topic from cloud storage. In case of failure, the reason for the failure appears as `ERROR` lines in the Redpanda application log. For each topic, this reports errors for all partitions, but for each partition, only the first error is reported.
```